Index: src/asset/javascript/site.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/asset/javascript/site.js	(revision )
+++ src/asset/javascript/site.js	(revision )
@@ -12,64 +12,68 @@
 
 // variables to show/hide functions to the about-section
     let showAbout = function () {
+        body.addClass("about-bg");
+        about.addClass("active");
         setTimeout(function () {
-            $(".about-info").slideDown()
+            about.slideDown()
         }, 350);
-        $(".about-info").addClass("active");
-        $("body").addClass("about-bg");
     };
     let hideAbout = function () {
-        $("body").removeClass("about-bg");
-        $(".about-info").removeClass("active");
-        $(".about-info").slideUp();
+        body.removeClass("about-bg");
+        about.removeClass("active");
+        about.slideUp();
     };
-// variables to show/hide functions to the work-section
+//  variables to show/hide functions to the work-section
     let showWork = function () {
+        body.addClass("work-bg");
+        work.addClass("active");
         setTimeout(function () {
-            $(".work-info").slideDown();
+            work.slideDown();
         }, 350);
-        $(".work-info").addClass("active");
-        $("body").addClass("work-bg");
     };
     let hideWork = function () {
-        $("body").removeClass("work-bg");
-        $(".work-info").removeClass("active");
-        $(".work-info").slideUp();
+        body.removeClass("work-bg");
+        work.removeClass("active");
+        work.slideUp();
     };
-// variables to show/hide functions to the skills-section
+//  variables to show/hide functions to the skills-section
     let showSkills = function () {
+        body.addClass("skills-bg");
+        skills.addClass("active");
         setTimeout(function () {
             skills.slideDown();
         }, 350);
-        skills.addClass("active");
     };
     let hideSkills = function () {
+        body.removeClass("skills-bg");
         skills.removeClass("active");
         skills.slideUp();
     };
-// variables to show/hide functions to the contact-section
+//  variables to show/hide functions to the contact-section
     let showContact = function () {
+        body.addClass("contact-bg");
+        contact.addClass("active");
         setTimeout(function () {
             contact.slideDown();
         }, 350);
-        $(".contact-info").addClass("active");
     };
     let hideContact = function () {
+        body.removeClass("contact-bg");
         contact.removeClass("active");
         contact.slideUp();
     };
 
 // When the about-circle is clicked, the content is shown. If another circle is active, this will not be shown.
-    $(".about-info").click(function () {
-        if ($(".work-info").hasClass("active work-bg")) {
+    about.click(function () {
+        if (work.hasClass("active") && (body.hasClass("work-bg"))) {
             hideWork();
             showAbout();
         }
-        else if (skills.hasClass("active skills-bg")) {
+        else if (skills.hasClass("active") && (body.hasClass("skills-bg"))) {
             hideSkills();
             showAbout();
         }
-        else if (contact.hasClass("active contact-bg")) {
+        else if (contact.hasClass("active") && (body.hasClass("contact-bg"))) {
             hideContact();
             showAbout();
         }
@@ -79,16 +83,16 @@
     });
 
 // When the work-circle is clicked, the content is shown. If another circle is active, this will not be shown.
-    $(".work-info").click(function () {
-        if ($(".about-info").hasClass("active about-bg")) {
+    work.click(function () {
+        if (about.hasClass("active") && (body.hasClass("about-bg"))) {
             hideAbout();
             showWork();
         }
-        else if (skills.hasClass("active skills-bg")) {
+        else if (skills.hasClass("active") && (body.hasClass("skills-bg"))) {
             hideSkills();
             showWork();
         }
-        else if (contact.hasClass("active contact-bg")) {
+        else if (contact.hasClass("active") && (body.hasClass("contact-bg"))) {
             hideContact();
             showWork();
         }
@@ -99,15 +103,15 @@
 
 // When the skills-circle is clicked, the content is shown. If another circle is active, this will not be shown.
     skills.click(function () {
-        if (about.hasClass("active about-bg")) {
+        if (about.hasClass("active") && (body.hasClass("about-bg"))) {
             hideAbout();
             showSkills();
         }
-        else if (work.hasClass("active work-bg")) {
+        else if (work.hasClass("active") && (body.hasClass("work-bg"))) {
             hideWork();
             showSkills();
         }
-        else if (contact.hasClass("active contact-bg")) {
+        else if (contact.hasClass("active") && (body.hasClass("contact-bg"))) {
             hideContact();
             showSkills();
         }
@@ -118,15 +122,15 @@
 
 // When the contact-circle is clicked, the content is shown. If another circle is active, this will not be shown.
     contact.click(function () {
-        if (about.hasClass("active about-bg")) {
+        if (about.hasClass("active") && (body.hasClass("about-bg"))) {
             hideAbout();
             showContact();
         }
-        else if (work.hasClass("active work-bg")) {
+        else if (work.hasClass("active") && (body.hasClass("work-bg"))) {
             hideWork();
             showContact();
         }
-        else if (skills.hasClass("active contact-bg")) {
+        else if (skills.hasClass("active") && (body.hasClass("skills-bg"))) {
             hideSkills();
             showContact();
         }
